### PR TITLE
require pylama as part of travis

### PR DESCRIPTION
**Pull request by ktbyers**
travis-ci must pass before merging this change.

### CODE REVIEW

**Comment by ktbyers** [comment]
Okay, this should be ready.

**Comment by mirceaulinic** [comment]
As per https://github.com/napalm-automation/napalm-ios/issues/22, `requirements-dev.txt` should contain

``` yaml
pytest
pytest-cov
pytest-json
pytest-pythonpath
pylama
flake8-import-order
-r requirements.txt
```

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ python:
 - 2.7
 install:
 - pip install -r requirements.txt
```
Comment (mirceaulinic) [inline .travis.yml]: If requirements-dev.txt has the content specified in https://github.com/napalm-automation/napalm-ios/issues/22, can remove this line

**Comment by ktbyers** [comment]
Yes, I haven't converted over to David's new testing framework...so most of those things are unneeded until we do.

**Comment by ktbyers** [comment]
Let me update with setup.cfg instead of pylama.ini

**Comment by ktbyers** [comment]
Okay, this is broke because of last commit...fixing it.

**Comment by ktbyers** [comment]
@mirceaulinic Okay, I think this is correct now per the standard referenced in #22

**Review by mirceaulinic — APPROVED** [review]
+1

**Pull request merged**